docs/administration/configuration.md

```diff
@@ -28,7 +29,19 @@ security:
     cert_file: "" # The optional location of an SSL cert to use
     key_file: "" # The optional location of an SSL key to use
 locket:
-    token: extremely secure password # The password to use to secure the `/lockets` endpoint
+    token: extremely secure password # The password used to secure the `/lockets` endpoint
 prometheus:
-    token: extremely secure password # The password to use to secure the `/metrics` endpoint
+    token: extremely secure password # The password used to secure the `/metrics` endpoint
+```
+
+### locketd.yml
+```yaml
+environment: release # Enables verbose logging (debug or release)
+port: 8000 # Port for locketd
+security:
+    rh4NaXhju914cn60CHmuMREeQG1Qdh53o4sQ9iZWVlA= # A secure 32 byte key; try `openssl rand -base64 32`
```

Comment:
Maybe specify that it has to be the same as Cell's.

Reply:
bruh wait

Reply:
`security` also isn't a string; you want `security.secret`.